### DISCUSSION TITLE
how to get logs from windows, when firewall all logs send pc, port 9001

### DISCUSSION THREAD

**Discussion by sevadamuradyan** [Security-Onion-Solutions/securityonion#12654] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

12

### Storage for /

256

### Storage for /nsm

1100000

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

our network firewall log is coming to my computer how can I send sec-onion?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Have you installed and Elastic agent to pick up the logs? https://docs.securityonion.net/en/2.4/elastic-agent.html
